scripts/build.ps1

```diff
@@ -530,6 +530,9 @@ function Create-NugetPackages
     # Pass Newtonsoft.Json version to nuget pack to keep the version consistent across all nuget packages.
     $JsonNetVersion = ([xml](Get-Content $env:TP_ROOT_DIR\scripts\build\TestPlatform.Dependencies.props)).Project.PropertyGroup.JsonNetVersion
 
+    # In CI build "Package: TestPlatform SDK" task, "Nuget.exe pack" required JsonNetVersion property for creating nuget package.
+    Write-Host "##vso[task.setvariable variable=JsonNetVersion;]$JsonNetVersion"
```

Comment:
The knowledge of which CI we're executing in is now coming inside the build script. Is there a way to achieve this without the coupling?

Reply:
Added separate script `vsts-prebuild.ps1`.